client/reader/lib/site-description-blacklist/test/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { isSiteDescriptionBlacklisted } from '../';
+
+describe( 'isSiteDescriptionBlacklisted', () => {
+	it( 'should return true if a site description is blacklisted', function() {
+		const blackListedDescription = 'Just another WordPress site';
+		expect( isSiteDescriptionBlacklisted( blackListedDescription ) ).to.equal( true );
```

Comment:
No strong prefs there really - but`.to.be.true` is more concise, so I'll switch it :)